specs/stork-deployment.yaml

```diff
@@ -110,7 +110,7 @@ spec:
         # automatically update schedulerName
         #- --app-initializer=true
         imagePullPolicy: Always
-        image: openstorage/stork:2.2.4
+        image: DEV_STORK_IMAGE 
```

Comment:
Let's not add placeholders to the specs, someone should be able to apply these specs directly

Reply:
Removed placeholders.